apps/nextjs-website/src/helpers/metadata.helpers.ts

```diff
@@ -13,6 +14,15 @@ type MakeMetadataParams = {
 
 type MakeMetadataFunction = (params: MakeMetadataParams) => Metadata;
 
+export const makeMetadataFromStrapi = (seo: SEO): Metadata => {
+  return makeMetadata({
+    title: seo.metaTitle,
+    description: seo.metaDescription,
+    url: seo.canonicalURL,
+    image: seo.metaImage?.data?.attributes?.url,
```

Comment:
You should add more properties if you want to use the preexistent `makeMetadata` function.
The home page should show the results of all of the SEO properties